helpers/strReplace.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const factory = () => {
+    return function(str, substr, newSubstr, iteration) {
+        if (typeof str !== 'string' || typeof substr !== 'string' || typeof newSubstr !== 'string') {
```

Comment:
In case you want to use this helper with concat or replace helper this condition won't pass. 
To unlock that possibility, please use https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/lib/common.js#L12 before type checking.